braket_features/Getting devices and checking device properties.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting a device and checking device properties\n",
```

Comment:
copy-pasted from earlier comment:
"Getting devices" then. I'll point out the existing documentation already uses "retrieve": https://docs.aws.amazon.com/braket/latest/developerguide/braket-using-boto3-client.html

Reply:
I agree. It is the problem with my NBI. I changed it but the update doesn't get applied.